DeveloperDocumentation/Deployment/deployment-local.md

```diff
@@ -62,7 +62,24 @@ curl -H "Authorization: bearer <id token>" http://localhost:8091/whoami
 and some details about that user should be shown.
 
 ### Prepare authentication for fetching GitHub packages
-https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages
+1. Navigate to the root directory of your machine, then do the following:
+    ```shell
+    ls -a
+    ```
+
+2. You should see a .m2 directory. Navigate to it
```

Comment:
Also might be worth mentioning that if they do not see the folder, to create it.